python/feathub/examples/nyc_taxi_spark_client.py

```diff
@@ -24,7 +24,10 @@
 def main() -> None:
     client = FeathubClient(
         props={
-            "processor": {"type": "spark", "spark": {"master": "local[1]"}},
+            "processor": {
+                "type": "spark",
+                "spark": {"master": "spark://localhost:7077"},
+            },
```

Comment:
Let's modify `run-examples.sh` and relevant CI files if any, and test that this example can be executed correctly by manually triggering the CI workflow.